src/App.tsx

```diff
@@ -19,6 +20,7 @@ const intersectionCallback = (entries: IntersectionObserverEntry[]) => {
 
 export const App = () => {
   const [showOnboarding, setShowOnboarding] = useState(true)
+  const [showAuth, setshowAuth] = useState(true)
```

Comment:
The Auth Modal should stay hidden by default and only show when the user click the profile avatar or tries to use a feature that requires login.